devito/operator/operator.py

```diff
@@ -1020,6 +1025,11 @@ def apply(self, **kwargs):
 
     # Performance profiling
 
+    def _emit_args_profiling(self, tag=''):
```

Comment:
honestly I would aggregate everything inside `_emit_apply_profiling`. I think what you're doing here is very important so I'd try to get it "ideal" from the get go. How about something along these lines? I'd certainly find it a bit less verbose and easily extensible 

```
Operator XXX ran in ....
Global performance ...
Global performance w/o setup
Excluded Python-level overheads: 2.2 s [1.5 s args pre-proc, 0.5 s args post-proc]
```

Either "Excluded" or "Additional" or perhaps just nothing...

Anyway, food for thought